pkg/util/util.go

```diff
@@ -522,3 +524,18 @@ func IsGlobExpMatch(strToMatch string, globExps []string) (bool, error) {
 	}
 	return false, nil
 }
+
+// PrintMachineOutput prints json output
+func PrintMachineOutput(outputFlag string, resource interface{}) (string, error) {
```

Comment:
@surajnarwade Please add coverage for this function

Reply:
Added

Reply:
This function shouldn't be named `Print..` as it is not printing anyhting

Reply:
> @surajnarwade Please add coverage for this function

I think somehow you missed to add UTs for function ```CheckOutputFlag``` and ```MachineOutput```